lib/libvcc/vmodtool.py

```diff
@@ -1180,6 +1181,20 @@ def json(self, fo, fnx):
         fo.write('\t\"\\n\\x03\"\n};\n')
         fo.write('#undef STRINGIFY\n')
 
+    # from varnish-cache include/generate.py
+    def version(self):
+        srcdir = os.path.dirname(self.inputfile)
+
+        for pkgstr in open(os.path.join(srcdir, "Makefile")):
+            if pkgstr[:14] == "PACKAGE_STRING":
+                break
+        pkgstr = pkgstr.split("=")[1].strip()
+
+        gitver = subprocess.check_output([
+            "git -C %s rev-parse HEAD 2>/dev/null || echo NOGIT" %
+            srcdir], shell=True, universal_newlines=True).strip()
+
+        return pkgstr + " " + gitver
```

Comment:
We shouldn't ship that to third-party VMOD authors, this is very specific to this repository.